age/Cargo.toml

```diff
@@ -46,6 +46,7 @@ bech32 = "0.8"
 # - RSAES-OAEP from RFC 8017 with SHA-256 and MGF1
 num-traits = { version = "0.2", optional = true }
 rsa = { version = "0.5", optional = true }
+pkcs1 = "0.2.3"
```

Comment:
Note that the `rsa` crate re-exports this as `rsa::pkcs1`, so you don't need to include it separately here

Reply:
Hi @tarcieri,

I was unsure of Rust conventions and style here.

My personal take was to ‘flatten in the dependency tree’ in the sense that if one directly references a feature of a transitive dependency, then one should turn that into a direct dependency declaration in the build. But then Rust’s language feature of ‘re-export’ sets it apart from other languages I am more familiar with.

To be true to my aesthetic sense, I suppose I should have used
```rust
use pkcs1::FromRsaPrivateKey;
```
rather than
```rust
use rsa::pkcs1::FromRsaPrivateKey;
```

Reply:
In the RustCrypto crates we do these sorts of re-exports to simplify upgrades. We frequently make breaking changes, but if e.g. `pkcs1` is only used by the `rsa` crate, then you only have to worry about bumping the `rsa` crate's version and don't have to worry about properly coordinating `pkcs1` vs `rsa` versions.

My recommendation would be to only pull in `pkcs1` directly if you are pulling in more than one crate which uses the `pkcs1` traits, or need to activate a feature which you can't activate transitively through the `rsa` crate. AFAICT that doesn't seem to be the case here.

Reply:
ok, thanks for the explanation;
removed in
https://github.com/str4d/rage/pull/261/commits/2f3cd681a8e4c91ed9ba45b5bfca5f683c9cd18b